docs/.vuepress/public/schema.yml

```
@@ -0,0 +1,401 @@
+openapi: '3.0.2'
+info:
+  title: Web3.Storage API
+  description: |
+    This documentation describes the HTTP API for [**Web3.Storage**](https://web3.storage), which allows you to quickly and easily build applications using decentralized data storage persisted by [Filecoin](https://filecoin.io) and available over [IPFS](https://ipfs.io).
+    
+    You can also interact with this API using client libraries. See [**docs.web3.storage**](https://docs.web3.storage) for more information on using a client library, as well as a variety of other guides and how-tos for Web3.Storage as a whole.
+  version: '1.0'
+
+servers:
+  - url: https://api.web3.storage
+
+tags:
+  - name: Web3.Storage HTTP API
+paths:
+  /upload:
+    post:
+      tags:
+        - Web3.Storage HTTP API
+      summary: Upload and store one or more files
+      description: |
+        Store files using Web3.Storage. You can upload either a single file or multiple files.
+
+        Send the POST request with one of the following:
+        - a single file, with a single blob/file object as the body
+        - multiple files, as `FormData` with `Content-Disposition` headers for each part to specify filenames and the request header `Content-Type: multipart/form-data`
+Requests to this endpoint have a maximum payload size of 100MB. To upload larger files, see the documentation for the `/car` endpoint.
```

Needs indent to validate.

```suggestion

        Requests to this endpoint have a maximum payload size of 100MB. To upload larger files, see the documentation for the `/car` endpoint.
```